hunting/README.md

```diff
@@ -0,0 +1,54 @@
+- **TOML Fields**: To ensure the hunt queries are consistent and comprehensive, it's important to structure the threat detection rules with specific fields. When contributing a new rule, please include the following fields in the TOML file to describe and configure the analytic:
+  - **author**: The name of the individual or organization authoring the rule.
+  - **integration**: The specific integration or data source the rule applies to, such as `aws_bedrock.invocation`.
+  - **uuid**: A unique identifier for the rule to maintain version control and tracking.
+  - **hypothesis**: A brief description of what the rule is intended to detect or hypothesize about potential threats.
+  - **name**: A descriptive name for the rule that clearly indicates its purpose.
+  - **analytic**: The actual query or analytic expression written in the appropriate query language that executes the detection logic.
+  - **notes**: An array of strings providing detailed insights into the rationale behind the rule, suggestions for further investigation, and tips on distinguishing false positives from true activity.
+  - **mitre**: Reference to applicable MITRE ATT&CK tactics or techniques that the rule addresses, enhancing the contextual understanding of its security implications.
+  - **references**: Links to external documents, research papers, or websites that provide additional information or validation for the detection logic.
+
+- **Documentation (Optional)**: Include a `README.md` in each subfolder describing the queries and their purposes. This would include a brief description of the new category.
+
+### Field Usage
+Use standardized fields where possible to ensure that queries are compatible across different data environments and sources.
+
+### Review and Pull Requests
+Follow the standard [contributing guide](../CONTRIBUTING.md). Please remember to use the generate_markdown.py script to update the documentation after adding or updateing queries.
+
+## Using the Script to Generate Markdown
+
+The `generate_markdown.py` script is provided to automate the creation of Markdown files from TOML rule definitions. Here’s how to use it:
+
+- **Generating Markdown**: Run `python generate_markdown.py` from the root of the `hunting` directory. This will generate Markdown files for each TOML file and update the `index.md` to include links to the new Markdown files.
+- **Structure**: Rules should be written in TOML and saved under the `rules` directory. The script will automatically convert them into Markdown and save them in the `docs` directory within the respective category folder.
+
+### Sample Directory Structure Example
+
+```config
+.
+├── README.md
+├── generate_markdown.py
+├── index.md
```

Comment:
any reason to not also name these `readme.md` for consistency with the rest of the repo?

Reply:
Our other README.md files are capitalized. Are you saying you want them all lowercased?

Reply:
disregard